mmv1/third_party/terraform/tests/resource_dns_record_set_test.go.erb

```diff
@@ -493,20 +527,56 @@ resource "google_dns_record_set" "foobar" {
 
   routing_policy {
     wrr {
-      weight       = 0.9
-      rrdatas      = ["127.0.0.1"]
+      weight  = 0
+      rrdatas = ["1.2.3.4", "4.3.2.1"]
     }
+
+    wrr {
+      weight  = 0
+      rrdatas = ["2.3.4.5", "5.4.3.2"]
+    }
+
     wrr {
-      weight       = %g
-      rrdatas      = ["%s"]
+      weight = 1.0
+
+      health_checked_targets {
+        internal_load_balancers {
+          load_balancer_type = "regionalL4ilb"
+          ip_address         = google_compute_forwarding_rule.default.ip_address
+          port               = "80"
+          ip_protocol        = "tcp"
```

Comment:
This should really be `google_compute_forwarding_rule.default.project.ip_protocol`, but because of the difference in casing, I think users will need to specify it explicitly like this :/